### DISCUSSION TITLE
RFC-01: Language on our website

### DISCUSSION THREAD

**Discussion by Norbiros** [Gwardia-Czapli/GwardiaHub#17] [RFC] [answered]
There are 3 different solutions:

1. Implement English universally, but certain content, like data from the mobidziennik calendar, might not fit.
2. Use Polish everywhere
3. Use Django's i18n for supporting multiple languages, but we need to be aware that maintaining two languages effectively will require a significant time investment.

**Comment by goteusz-maszyk** [comment, marked as answer]
Polish
